src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/UserInfoController.scala

```diff
@@ -15,19 +15,28 @@
 
 package za.co.absa.hyperdrive.trigger.api.rest.controllers
 
+import org.springframework.beans.factory.annotation.Value
 import za.co.absa.hyperdrive.trigger.models.UserInfo
 import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.security.core.userdetails.UserDetails
 import org.springframework.web.bind.annotation._
 
 @RestController 
 class UserInfoController {
+  @Value("${environment:Unknown}")
+  val environment: String = ""
+  @Value("${version:Unknown}")
+  val version: String = ""
 
   @GetMapping(path = Array("/user/info"))
   def userInfo(): UserInfo = {
     val auth = SecurityContextHolder.getContext.getAuthentication
     val principal = auth.getPrincipal.asInstanceOf[UserDetails]
-    UserInfo(principal.getUsername)
+    UserInfo(
+      username = principal.getUsername,
```

Comment:
I think environment and version don't belong to userinfo. It has nothing to do with the user. Also, in principle, env and version should be displayed even if the user is logged out

Reply:
I am going to rewrite it